### ISSUE TITLE
Can you add a file name as a description of the picture?

### ISSUE THREAD

**Issue by zjlx** [holgerbrandl/pasteimages#8]
Can you add a file name as a description of the picture by default?
For example:! [Filename] (/ path / filename.png)
Thank you very much！

**Comment by holgerbrandl** [comment]
Having a static 'filename' won't work because we need a different name for each image. That's why it's using checksums as a default, but you're free to name it differently in the dialog.

Does this answer your question?

**Comment by zjlx** [comment]
Oh no, I mean, when I am in the dialog box enter the file name .Can you add the input text as  the alt attribute text for the image?
Thanks for your reply!

**Comment by holgerbrandl** [comment]
Like in `![fancy pic](.notes_images/fancy_pic.png)`? This should be easy I guess, but we should make it an opt-in preference, because I imagine most users to prefer the more simplistic insert without alt text.

**Comment by zjlx** [comment]
Yes, that's right. What you think is right, and I also think that a choice should be added, not a default, and the effect will be different for different people.
